packages/icons/src/trimet-mode-icon-2021.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Bicycle, Car, Micromobility, Walk } from "./trimet-2021";
+
+/**
+ * These icons are not an entire set and are only used in the 2021
+ * custom TriMet mode selector component
+ */
+function TriMetModeIcon({ mode, ...props }) {
```

Comment:
Probably rename to `TriMetModeIcon2021` to avoid ambiguity.

Reply:
This was done intentionally to prevent both icon sets from being used at the same time.